src/time.cr

```diff
@@ -524,8 +524,10 @@ struct Time
     utc(seconds: seconds, nanoseconds: nanoseconds.to_i)
   end
 
-  def self.now(*args)
-    {% raise "`Time.now` has been removed in Crystal 0.28.0, use `Time.local` or `Time.utc` instead.\n\nMore information on this change: https://github.com/crystal-lang/crystal/issues/5346" %}
+  # DEPRECATED: `Time.now` is deprecated, use `Time.local` or `Time.utc` instead.
+  # More information on this change: https://github.com/crystal-lang/crystal/issues/5346
+  def self.now
+    local
```

Comment:
This is not making an alias of the other overloads of `Time.local`. `*args` and `**kwargs` should be forwarded.

Reply:
Ooops.